### PR TITLE
Feature/1

### PR DESCRIPTION
**Pull request by Hawawou**
this PR adds Mendolearn, Colorfol and Dastudy to the catalogue

related to this issue #1

### CODE REVIEW

File: src/res/apps.yaml [hunk 1]
```diff
@@ -194,3 +194,50 @@ items:
...
+    #twitter group handle
+    twitter: https://twitter.com/mendolearn_cam
+    #facebook link
+    facebook: https://web.facebook.com/mendolearn
+    #android app store
+    playstore: https://play.google.com/store/apps/details?id=com.mendolearn&hl=fr&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1
+    
+    image: mendo_learn.png
+
+    #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:214]: This comments should not be indented

File: src/res/apps.yaml [hunk 2]
```diff
@@ -194,3 +194,50 @@ items:
...
+    
+    twitter: https://twitter.com/ColorfolApp?s=20
+    #facebook link
+    facebook: https://web.facebook.com/ColorfolApp
+    #android app store
+    playstore: https://play.google.com/store/apps/details?id=com.dev.mobile.colortech
+    
+    image: colorfol.jpg
+
+       #--------------------------------------------------------------
```
Comment (elhmn) [inline src/res/apps.yaml:232]: ditto (indentation)

File: src/res/apps.yaml [hunk 3]
```diff
@@ -194,3 +194,50 @@ items:
...
+    category: platform
+    #max of five tags
+    tags:
+      - education
+    description: |
+        Dastudy offers you easy access to knowledge and entertainment thanks to PDF documents, software, video games, video tutorials
+    twitter: https://twitter.com/dastudy19?s=20
+    website: https://www.dastudy.net/
+    
+    image: dastudy.jpg
```
Comment (elhmn) [inline src/res/apps.yaml:243]: Can you add the comment bellow at the bottom of this file ?
```yaml
#--------------------------------------------------------------
```

**Review by elhmn — COMMENTED** [review]
Nice one @Hawawou , I just left few comments regarding the indentation 👍

**Comment by Hawawou** [comment]
I have modified them.

**Comment by elhmn** [comment]
Great stuff @Hawawou , I will merge it now 🚀

**Pull request merged**